tests/PolyType.Tests/IsRequiredTests.cs

```diff
@@ -0,0 +1,188 @@
+﻿using PolyType;
+using PolyType.Tests;
+
+namespace PolyType.Tests;
+
+public abstract partial class IsRequiredTests(ProviderUnderTest providerUnderTest)
+{
+    [Fact]
+    public void UnattributedProperties()
```

Comment:
Would it be possible to include a few types that set the new properties in PolyType.TestCases as well?

Reply:
Done